bin/backup.sh

```diff
@@ -18,17 +18,13 @@ if [[ -z "$S3_BUCKET_PATH" ]]; then
   exit 1
 fi
 
-#install aws-cli
-curl https://s3.amazonaws.com/aws-cli/awscli-bundle.zip -o awscli-bundle.zip
-unzip awscli-bundle.zip
-chmod +x ./awscli-bundle/install
-./awscli-bundle/install -i /tmp/aws
+DATABASE_NAME="$(/app/vendor/heroku-toolbelt/bin/heroku pg:info -a $APP | grep DATABASE_URL | awk '{split($0,a," "); print a[2]}')"
```

Comment:
I need to be able to set my DATABASE_NAME as i do not do a backup of the DATABASE_URL.  As mentioned, we are running our backups from a completely separate heroku application.